src/mmw/js/src/compare/views.js

```diff
@@ -164,14 +152,77 @@ var CompareScenariosView = Marionette.CompositeView.extend({
     }
 });
 
-var CompareChartView = Marionette.ItemView.extend({
-    template: compareChartTmpl,
+var CompareModelingView = Marionette.LayoutView.extend({
+    //model: modelingModels.ScenarioModel
```

Comment:
No, I've been putting the model property as a comment in views as a reminder of the type of model associated with a view. If you leave it uncommented, it sometimes creates a problem, although I can't remember when/why. Also, having it as a comment makes sense because that's what it essentially is, since the property is overridden by what is passed to the constructor.